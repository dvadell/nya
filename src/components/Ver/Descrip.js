import React from 'react'

const Descrip = ({descr, fechahora}) => {
    let fechaLinda = '(Sin fecha)'
    if (fechahora) {
        let fecha = fechahora.slice(0,10); 
        let hora  = fechahora.slice(11,16);
        fechaLinda = fecha + ' ' + hora + 'hs';
    }
    
    return descr ? (
            <div>
                <h3>Descripci√≥n</h3>
                <p><strong>{fechaLinda}:</strong>  {descr}</p>
            </div>
    ) : null
}

export default Descrip;
