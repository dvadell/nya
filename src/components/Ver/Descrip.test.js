import { shallow } from 'enzyme'
import React from 'react';
import Descrip from './Descrip'

let gato = {
    coords: "-34.5473024, -58.451558399999996",
    descr: "",
    fechahora: "2019-06-21T22:37:51.154Z",
    img: "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD",
    nombre: "",
    __v: 0,
    _id: "5d0d5c3f4c2c1600275a0692"
}

describe('description', () => {
    it('expect to render Descripción component', () => {
    expect(shallow(<Descrip descr={gato.descr}
                            fechahora={gato.fechahora} />)).toMatchSnapshot()
    })
    
    it('expect to render empty Descripción component', () => {
        expect(shallow(<Descrip />)).toMatchSnapshot()
    })
})

