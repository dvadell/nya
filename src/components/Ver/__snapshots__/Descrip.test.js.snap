// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`description expect to render Descripción component 1`] = `ShallowWrapper {}`;

exports[`description expect to render empty Descripción component 1`] = `ShallowWrapper {}`;
